pages/add/add.scss

```diff
@@ -3,6 +3,12 @@
     margin-top: $form-field-vertical-margin / 2;
   }
 
+  fieldset.title {
+    .error {
+      margin-right: 50px;
```

Comment:
Let's use `em` here, since this margin is specifically to pad for text on the right side. `3em` looks good.

Reply:
Do we have convention about when to use `px`, `rem`, and `em`? I've been using `rem` for mostly font sizes and `px` for spacing. I almost never used `em`...

Reply:
I don't think we do, though we probably should (cc @gvn). You could go with rem here, based on the size of the text being used for the character count.